packages/config/__test__/Config.test.js

```
@@ -1,10 +1,8 @@
-import Joi from 'joi';
 import { Config } from '../src/Config.js';
+import Joi from 'joi';
 
 describe('Config - create config with schema', () => {
   it('should throw an error if no schema', () => {
```

should throw an error if not joi type schema